groovin-music/src/components/Home/Homepage.js

```diff
@@ -20,11 +20,11 @@ export default function Homepage() {
       </div>
 
       <div className="main">
-        <Route exact path="/auth/spotify/callback" component={HomeMain} />
-        <Route path="/auth/spotify/callback/playlist-page" component={PlaylistPage} />
-        <Route path="/auth/spotify/callback/discover-page" component={DiscoverPage} />
-        <Route path="/auth/spotify/callback/player-page" component={PlayerPage} />
-        <Route path="/auth/spotify/callback/settings-page" component={SettingsPage} />
+        <Route exact path="/" component={HomeMain} />
```

Comment:
great!  maybe this could be cleaned up in the future by creating a Routes folder for these?  this works, so that's all that matters!

Reply:
I would clean these up in Canvas 2. We don't need all the extraneous stuff...just the stuff to right of `callback`.